### PR TITLE
Documentation clarity & less docker layers

### PR DESCRIPTION
**Pull request by pastjean**
Reduced the number of docker commands to make it a little faster, it generates less layers and rebuilds faster that way.

I also rephrased part of the readme and added the instructions on how to run it without docker since people that do not know docker or kotlin or jvm world seems to pass a lot of time. Instructions on how to build/run also are helpful.

### CODE REVIEW

File: docker-start.sh [hunk 1]
```diff
@@ -5,10 +5,8 @@ docker build . --tag pleo-antaeus
 
 # Build the code.
 docker run \
-  --name pleo-anteus-app \
   --publish 7000:7000 \
   --rm \
   --interactive \
   --tty \
-  --volume pleo-antaeus-build-cache:/home/pleo/.gradle \
```
Comment (pastjean) [inline docker-start.sh]: ill readd this with the good path

File: README.md [hunk 2]
```diff
@@ -1,53 +1,78 @@
...
+
+*Running through docker*
+
+Install docker for your platform
+
+```
+make docker-run
+```
+
+### App Structure
```
Comment (mapoulin) [inline README.md]: I would revert this whole section, I don't think this adds any clarity. In fact, some of the things are harder to read.

File: README.md [hunk 3]
```diff
@@ -1,53 +1,78 @@
...
-Happy hacking 😁!
-
-## How to run
-```
-./docker-start.sh
-```
-
-## Libraries currently in use
+### Main Libraries currently in use
+* [Sqlite3](https://sqlite.org/index.html) - Database storage
```
Comment (mapoulin) [inline README.md]: I would revert this as well. SQLite isn't embedded in the code, we use it as a dependency, such as the JVM, Docker, etc.

File: README.md [hunk 4]
```diff
@@ -1,53 +1,78 @@
...
+
+### Building
+
+```
+./gradlew build
+```
+
+### Running
+
+There are 2 options for running anteus. You either need libsqlite3 or docker. Docker is easier but more invasive to setup.
```
[the remark this answers is not in the document]
Comment (pastjean) [reply README.md]: People really get mixed up with docker. Learning kotlin, docker , ... they spend too much time on this.

File: README.md (hunk 3 above)
Comment (pastjean) [reply README.md]: I'm not sure i do agree, it is a dependency and a library. You will code queries differently depending on the database engine you have here. I'll put it last. I remember Alexey telling me he'd have used different styles with postgres

File: README.md (hunk 2 above)
Comment (pastjean) [reply README.md]: which whole section

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (pastjean) [reply README.md]: I've put it first though and added that we recommend this one

**Comment by pastjean** [comment]
@mapoulin most comments adressed

**Pull request merged**